chainer/functions/array/spatial_transformer_grid.py

```diff
@@ -115,7 +112,7 @@ def _backward(self, inputs, grad_outputs):
         return gtheta,
 
 
-def spatial_transformer_grid(theta, output_shape, use_cudnn=True):
+def spatial_transformer_grid(theta, output_shape):
```

Comment:
Could you show an error message when this function is used with `use_cudnn` flag? See e.g. the implementation of `convolution_2d`.